src/test/java/wromaciej/moviesrental/movies/service/CartServiceTest.java

```diff
@@ -18,7 +19,8 @@
 public class CartServiceTest {
 
     private MovieRepository movieRepository = Mockito.mock(MovieRepository.class);
-    private CartService cartService = new CartService(movieRepository);
+    private MovieService movieService = new MovieService(movieRepository);
```

Comment:
You should mock MovieService now and remove MovieRepository completely from CartService. Why to depend from repository if you can depend only on MovieService?